tools/pontoon/resources.md

```diff
@@ -1,91 +1,97 @@
+
+In the `REVIEW TRANSLATIONS` section, the translator can approve suggestions or remove approved translations for all selected strings. Upon clicking `APPROVE ALL`, the text in the green button will be replaced with the number of new approved translations.
 
-In the window, the translator can approve or remove the history of all selected strings int eh `REVIEW TRANSLATIONS` section. Upon clicking `APPROVE ALL`, the text in the green button will be replaced with the number of new approved translations.
+In the `REPLACE IN TRANSLATIONS` section, the translator can input a series of characters to search for, and a series of characters to use as a replacement. This is a basic find and replace feature that will work only on the selected strings.
 
-In the `REPLACE IN TRANSLATIONS` section, the translator can input a series of characters to search for and a series of characters to replace the first series with. This is a basic find and replace feature that will work only on the selected strings.
+Note that mass actions are only available to users with [translators rights](users.md#user-roles).
 
 ## Keyboard shortcuts
 
-While working in the translation workspace, keyboard shortcuts can be used for a mouseless translation experience. Click the keyboard icon in the translation workspace to show a menu of the keyboard shortcuts.
+While working in the translation workspace, keyboard shortcuts can be used for a mouseless translation experience. Click the keyboard icon in the translation workspace to show a complete list of available keyboard shortcuts.
 
 ![Keyboard shortcuts](/assets/images/pontoon/resources/shortcuts.png)
 
-## Download/upload translations
+## Download and upload translations
 
 Pontoon features the ability to download and upload translations, including translation memories. To access these features, click on your user icon in the top-right section of Pontoon. Note that you must be in the translation workspace for the download/upload options to display in the dropdown menu.
 
-![Profile dorpdown menu](/assets/images/pontoon/resources/profile_menu.png)
+![Profile dropdown menu](/assets/images/pontoon/resources/profile_menu.png)
 
-Note that translators can download a translation memory (.tmx) and translations (plain text file) or upload translations. Translators should communicate with their community or project coordinator to ensure that the file types are compatible with external software and that the edited files are compatible with Pontoon for uploading.
+Note that only translators can download a translation memory (.tmx) and translations, or upload translations. When downloading translations, the resource currently selected is downloaded in its original format, it’s not converted.
```

Comment:
I don't know how uploads are managed by pontoon. Is there existing documentation on that so I can read up?